src/types/build-tools.ts

```diff
@@ -0,0 +1,115 @@
+export interface BuildTool {
```

Comment:
Do we need these?

Reply:
No, I copied the entire `/types` folder in case we will need it for some component. I will remove unnecessary types files on the details page ticket so that I would know which files we don't need at all.